starknet_py/net/models/abi/definition_test.py

```diff
@@ -0,0 +1,334 @@
+import json
+from collections import OrderedDict
+
+import pytest
+
+from starknet_py.cairo.type_parser import UnknownCairoTypeError
+from starknet_py.net.models.abi.definition import AbiDefinition, AbiDecodingError
+from starknet_py.cairo.data_types import StructType, FeltType, ArrayType
+from starknet_py.tests.e2e.fixtures.misc import read_contract
+
+uint256_dict = {
+    "type": "struct",
+    "name": "Uint256",
+    "size": 2,
+    "members": [
+        # low and high were switched on purpose
```

Comment:
What's the purpose?

Reply:
Abi parser has to ensure the order as members have the offset field.

Reply:
Okay, I get it. Could this comment be improved? The test makes sense but it's not 100% obvious why it was switched up imo.